gobblin-core/src/test/java/gobblin/source/extractor/hadoop/AvroFsHelperTest.java

```diff
@@ -27,7 +27,7 @@
 
 public class AvroFsHelperTest {
 
-  @Test(expectedExceptions = IllegalArgumentException.class)
```

Comment:
Turns out the reason for this is: `For some reason, the jar hadoop-aws.jar which contains the implementation to NativeS3FileSystem is not present in the classpath of hadoop by default in the version 2.6.`.